x-pack/filebeat/input/netflow/netflow_test.go

```diff
@@ -60,6 +67,131 @@ type TestResult struct {
 	Flows []beat.Event `json:"events,omitempty"`
 }
 
+func newV2Context() (v2.Context, func()) {
+	ctx, cancel := context.WithCancel(context.Background())
+	return v2.Context{
+		Logger:      logp.NewLogger("netflow_test"),
+		ID:          "test_id",
+		Cancelation: ctx,
+	}, cancel
+}
+
+func TestNetFlow(t *testing.T) {
+	pcaps, err := filepath.Glob(filepath.Join(pcapDir, "*.pcap"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, file := range pcaps {
+		testName := strings.TrimSuffix(filepath.Base(file), ".pcap")
+
+		t.Run(testName, func(t *testing.T) {
+
+			configMap := conf.MustNewConfigFrom(mapstr.M{})
+			pluginCfg, err := conf.NewConfigFrom(configMap)
+			require.NoError(t, err)
```

Comment:
Same thing here, you don't need to create two configs.
```suggestion
			pluginCfg := conf.MustNewConfigFrom(mapstr.M{})
			netflowPlugin, err := Plugin(logp.NewLogger("netflow_test")).Manager.Create(pluginCfg)
			require.NoError(t, err)

```